app/models/concerns/taggable.rb

```diff
@@ -77,6 +77,34 @@ def freeform_string=(tag_string)
     parse_tags(Freeform, tag_string)
   end
 
+  def tag_string
```

Comment:
Can you move this method next to the other `_string` getters? It should also have a comment:

```
# Returns a string of tag names of all types
```